WangscapeTest/TestRasterImage.cpp

```diff
@@ -0,0 +1,105 @@
+#include <gtest/gtest.h>
+
+#include <noise/RasterImage.h>
+#include <noise/module/WangscapeModules.h>
+
+class TestRasterImage : public ::testing::Test
+{
+protected:
+    noise::module::GradientX gradientXModule;
+    noise::module::GradientY gradientYModule;
+    noise::module::Const constModule;
+    sf::Image image;
+    TestRasterImage()
+    {
+        constModule.SetConstValue(0.);
+    };
+    ~TestRasterImage() {};
```

Comment:
Do you need a destructor here? If yes (nothing comes to my mind 😕), use `= default` to keep the code consistent.